### PR TITLE
user/mu: new package

### PR DESCRIPTION
**Pull request by tjheeta**

### CODE REVIEW

File: user/mu/template.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+pkgname = "mu"
+pkgver = "1.12.6"
+pkgrel = 0
+build_style = "meson"
+hostmakedepends = ["meson", "pkgconf", "cmake", "emacs-console"]
```
Comment (nekopsykose) [inline user/mu/template.py:5]:
```suggestion
hostmakedepends = ["meson", "pkgconf", "emacs-console"]
```

not used

File: user/mu/template.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+makedepends = ["xapian-core-devel", "gmime-devel", "glib-devel"]
```
Comment (nekopsykose) [inline user/mu/template.py:6]: i think you also want guile-devel here

**Comment by nekopsykose** [comment]
the commit messages should be `user/mu: new package` not ` - `

File: user/mu/template.py (hunk 2 above)
Comment (nekopsykose) [reply user/mu/template.py:6]: you can then split usr/share/guile + usr/lib/guile to a subpackage

Comment (tjheeta) [reply user/mu/template.py:6]: hint on the correct code block to do it?

File: user/mu/template.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "pkgconf",
+]
+makedepends = [
+    "glib-devel",
+    "gmime-devel",
+    "xapian-core-devel",
+]
+pkgdesc = "Maildir-utils indexer/searcher and mu4e extension for emacs"
+maintainer = "tj <tjheeta@gmail.com>"
+license = "GPL-3.0-only"
```
Comment (nekopsykose) [inline user/mu/template.py:17]:
```suggestion
license = "GPL-3.0-or-later"
```

**Pull request merged**